girder/utility/assetstore_utilities.py

```diff
@@ -30,6 +31,16 @@
 }
 
 
+class GirderNoAssetstoreAdapterException(GirderException):
```

Comment:
This is an honest question: would `NoAssetstoreAdapterGirderException` make more sense as a name?